.storybook/preview-head.html

```diff
@@ -1,7 +1,7 @@
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
 <style>
     html, body {
-        font-family: "Font Awesome 5 Free";
+        font-family: "Lato";
```

Comment:
I think you can move this css rule to src/lib/index.css in this way all the uis importing core ui would benefit from it out of the box